<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>hotelmanage</class>
 <widget class="QDialog" name="hotelmanage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>675</width>
    <height>411</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;仿宋&quot;;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;请输入您的酒店名称：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>请输入酒店名称</string>
   </property>
  </widget>
  <widget class="QPushButton" name="roomsearchBtn">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>160</y>
     <width>141</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;仿宋&quot;;</string>
   </property>
   <property name="text">
    <string>查看客房</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ordersearchBtn">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>240</y>
     <width>141</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;仿宋&quot;;
font: 12pt &quot;仿宋&quot;;</string>
   </property>
   <property name="text">
    <string>查看订单</string>
   </property>
  </widget>
  <widget class="QPushButton" name="modifyBtn">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>320</y>
     <width>141</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 12pt &quot;仿宋&quot;;
</string>
   </property>
   <property name="text">
    <string>修改客房</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>30</y>
     <width>361</width>
     <height>331</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>请输入酒店密钥</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
